Update canceling-a-workflow.md (#30853)
github/docs: content/actions/managing-workflow-runs/canceling-a-workflow.md

1. To cancel the workflow run, the server re-evaluates `if` conditions for all currently running jobs. If the condition evaluates to `true`, the job will not get canceled. For example, the condition `if: always()` would evaluate to true and the job continues to run. When there is no condition, that is the equivalent of the condition `if: success()`, which only runs if the previous step finished successfully.
1. For jobs that need to be canceled, the server sends a cancellation message to all the runner machines with jobs that need to be canceled.

> 1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](../learn-github-actions/expressions#cancelled) especially for the cancel condition.

Author: ```suggestion
1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/en/learn-github-actions/expressions#cancelled) especially for the cancel condition.
```

> 1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/en/learn-github-actions/expressions#cancelled) especially for the cancel condition.

Author: ```suggestion
1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/en/actions/learn-github-actions/expressions#cancelled) especially for the cancel condition.
```

> 1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/actions/learn-github-actions/expressions#cancelled) especially for the cancel condition.

Reviewer: Suggesting the following, which is in line with our content model.
As for your question on the PR about links, you are correct, we need to use relative links 👍🏻 

```suggestion
1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use the `cancelled` expression to apply a  status check of `cancelled( )`. For more information see "[AUTOTITLE](/actions/learn-github-actions/expressions#cancelled)."
```

> 1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/en/actions/learn-github-actions/expressions#cancelled) especially for the cancel condition.

Author: ```suggestion
1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/actions/learn-github-actions/expressions#cancelled) especially for the cancel condition.
```

> 1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](https://docs.github.com/en/actions/learn-github-actions/expressions#cancelled) especially for the cancel condition.

Author: ```suggestion
1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](../learn-github-actions/expressions#cancelled) especially for the cancel condition.
```
